apps/condo/domains/billing/utils/unitName.utils.spec.js

```diff
@@ -0,0 +1,126 @@
+const  { normalizeUnitName } = require('./unitName.utils')
+
+const TEST_CASES = {
```

Comment:
As for me it looks like we can remove all `-` in results (optional)

Reply:
remove delimiter? can not understand why. If you mean normalizeUnitName fn result, some delimiter required to see, where is a word, where is a separate symbol